### PR TITLE
chore(docs): editorial review of sinks descriptions

### PR DESCRIPTION
**Pull request by maycmlee**
Editorial review of sinks descriptions to fit our [style guide](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md).

ref: DOCS-4685

### CODE REVIEW

File: src/kafka.rs [hunk 1]
```diff
@@ -57,9 +57,9 @@ pub struct KafkaAuthConfig {
 pub struct KafkaSaslConfig {
     /// Enables SASL authentication.
     ///
-    /// Only `PLAIN` and `SCRAM`-based mechanisms are supported when configuring SASL authentication via `sasl.*`. For
+    /// Only `PLAIN`- and `SCRAM`-based mechanisms are supported when configuring SASL authentication using `sasl.*`. For
     /// other mechanisms, `librdkafka_options.*` must be used directly to configure other `librdkafka`-specific values
-    /// i.e. `sasl.kerberos.*` and so on.
+    /// For example, `sasl.kerberos.*`.
```
Comment (spencergilbert) [inline src/kafka.rs:62]:
```suggestion
    /// for example, `sasl.kerberos.*`.
```

**Review by spencergilbert — APPROVED** [review]
Thank you!

File: src/kafka.rs [hunk 2]
```diff
@@ -57,9 +57,9 @@ pub struct KafkaAuthConfig {
 pub struct KafkaSaslConfig {
     /// Enables SASL authentication.
     ///
-    /// Only `PLAIN` and `SCRAM`-based mechanisms are supported when configuring SASL authentication via `sasl.*`. For
+    /// Only `PLAIN`- and `SCRAM`-based mechanisms are supported when configuring SASL authentication using `sasl.*`. For
     /// other mechanisms, `librdkafka_options.*` must be used directly to configure other `librdkafka`-specific values
-    /// i.e. `sasl.kerberos.*` and so on.
+    /// for example, `sasl.kerberos.*`.
```
[the remark this answers is not in the document]
Comment (maycmlee) [reply src/kafka.rs:62]: @spencergilbert would this be clearer?

> For other mechanisms, `librdkafka_options.*` must be used directly to configure other `librdkafka`-specific values. If using `sasl.kerberos.*` as an example, where the`*` could be `service.name`, `principal`, `kinit.md`, etc., then `librdkafka_options.*` would be `librdkafka_options.sasl.kerberos.service.name`, `librdkafka_options.sasl.kerberos.principal`, etc.

[the remark this answers is not in the document]
Comment (spencergilbert) [reply src/kafka.rs:62]: Yeah, seems fine to me!

[the remark this answers is not in the document]
Comment (maycmlee) [reply src/kafka.rs]: Thanks @spencergilbert, updated it!

**Review by spencergilbert — APPROVED** [review]

File: src/sinks/mezmo.rs [hunk 3]
```diff
@@ -75,20 +75,20 @@ pub struct MezmoConfig {
...
     #[configurable(metadata(docs::examples = "my-mac-address"))]
     mac: Option<String>,
 
-    /// The IP address that will be attached to each batch of events.
+    /// The IP address that is attached to each batch of events.
     #[configurable(metadata(docs::examples = "0.0.0.0"))]
     ip: Option<String>,
 
-    /// The tags that will be attached to each batch of events.
+    /// The tags that is attached to each batch of events.
```
Comment (neuronull) [inline src/sinks/mezmo.rs:91]:
```suggestion
    /// The tags that are attached to each batch of events.
```

File: src/sinks/kafka/config.rs [hunk 4]
```diff
@@ -104,7 +104,7 @@ pub struct KafkaSinkConfig {
 
     /// The log field name to use for the Kafka headers.
     ///
-    /// If omitted, no headers will be written.
+    /// If omitted, no headers is written.
```
Comment (neuronull) [inline src/sinks/kafka/config.rs:107]:
```suggestion
    /// If omitted, no headers are written.
```

File: src/sinks/honeycomb.rs [hunk 5]
```diff
@@ -26,7 +26,7 @@ pub struct HoneycombConfig {
     #[serde(skip, default = "default_endpoint")]
     endpoint: String,
 
-    /// The team key that will be used to authenticate against Honeycomb.
+    /// The team key that is used to authenticate against Honeycomb.
```
Comment (neuronull) [inline src/sinks/honeycomb.rs:29]: Unrelated to your change but "team" key seems weird to me. IDK much about Honeycomb but this seems like a typo to me "team" -> "API"

Comment (spencergilbert) [reply src/sinks/honeycomb.rs:29]: > X-Honeycomb-Team, which is your API key.

I think API is a good change, and this is why it got called team.

File: src/sinks/util/service.rs [hunk 6]
```diff
@@ -97,7 +97,7 @@ pub struct TowerRequestConfig {
 
     /// The time a request can take before being aborted.
     ///
-    /// It is highly recommended that you do not lower this value below the service’s internal timeout, as this could
+    /// Datadog highly recommends that you do not lower this value below the service's internal timeout, as this could
```
Comment (spencergilbert) [inline src/sinks/util/service.rs:100]: @maycmlee should this say Datadog here since it's shared between the DD docs and Vector?

Comment (spencergilbert) [reply src/sinks/util/service.rs:100]: Just wanted to confirm it was intentional, just caught me by surprise when I saw Datadog on all of the sinks

**Pull request merged**